docs/source/_notebooks/keras-text-classifiers.rst

```diff
@@ -0,0 +1,3685 @@
+        </p>
+    
+    
+        
+    
+        
+    
+        
+    
+        
+    
+    
+        
+    
+        
+    
+        
+    
+        
+    
+        
+    
+        
+    
+    
+        
+    
+        
+    
+        
+    
+        
+    
+        
+    
+        
+    
+    
+    
+
+
+
+
+As expected special words like ``<PAD>`` shouldn't have an effect on the
+explanation.
+
+Modify explanations with the ``relu`` and ``counterfactual`` arguments
+----------------------------------------------------------------------
+
+In the last section we only saw the "positive" words in our input, what
+made the class score "go up". To "fix" this and see the "negative" words
+too, we can pass two boolean arguments.
+
+``relu`` (default ``True``) only shows what makes the predicted score go
+up and discards the effect of counter-evidence or other classes in case
+of multiclass classification (set to ``False`` to disable).
```

Comment:
Maybe also would be good to add how it works inside, like "it discards negative gradients" or something similar to how it's described in the paper.